Ambiguous-Squids/Bangular
# Nc

# Description
Created shell Partial and controller for Creating a product page and linked it in the nav bar

# Related Ticket(s)
#13

## Steps to Test

* git fetch
* git checkout nc
* $ http-server
* http://localhost:8080/
* Click "Sell Products" in nav bar, make sure it takes you to the Sell Products partial view.
